### DISCUSSION TITLE
How to update form field when relation manager is updating?

### DISCUSSION THREAD

**Discussion by imknight** [filamentphp/filament#6181] [Help] [answered]
example I have order & order items relationship, so when I attach new order item in relationship manager, I would like the total field of the form is updated.

**Comment by danharrin** [comment, marked as answer]
You need to emit a Livewire event from the `after()` callback of the `AttachAction`. Then listen for the Livewire event on the edit page class of the resource, and call the magic `$refresh` method or a custom public method that contains your refreshing logic.
